src/components/test-slot/test-slot/test-slot.html

```diff
@@ -26,7 +28,8 @@
                     <ion-col size="2"></ion-col>
                     <ion-col size="7" class="exclamation-col align-center">
                         <indicators [showExclamationIndicator]="isIndicatorNeededForSlot() && !isTestCentreJournalADIBooking()"
-                                    [testStatus]="componentState.testStatus$ | async">
+                                    [testStatus]="componentState.testStatus$ | async"
+                                    [applicationId]="slot.booking.application.applicationId">>
```

Comment:
This has a double ```>```, think that will render in the template so can we remove one please